packages/host/app/services/card-service.ts

```diff
@@ -59,7 +59,6 @@ export default class CardService extends Service {
   @service private declare loaderService: LoaderService;
   @service private declare messageService: MessageService;
   private subscriber: CardSaveSubscriber | undefined;
-  private indexCards: Map<string, CardDef> = new Map();
```

Comment:
I dont know what the context of this is. It looks like it keeps `cards-grid` as an index card. But the issue is this  cached CardDef becomes unusable when accessing a symbol because we throw away the loader for this. Particularly, when using `getRealmURL`

Reply:
Owh I think cards-grid doesn't show in a non-operator-mode state

Reply:
it does--it's the cards-grid is our first (and only ) implementation of an index card. it's the grid of cards that you see when you are _not_ in operator mode (as well as the grid of cards that you see in operator mode.

The context around this is that the "copy cards" feature needs to know if the card that is being displayed in the stack is an index card, since only an index card can be the destination of a copy. This map is used to keep track of the index cards that the host has loaded.

Reply:
the answer here might be to use a different mechanism to determine index cards as part of card copy...  now that we have the realm URL header, that's probably a lot easier than when we first implemented this--the copy card button could leverage that instead of this cache...